test/memoryPoolAPI.cpp

```diff
@@ -124,10 +124,11 @@ INSTANTIATE_TEST_SUITE_P(
     mallocPoolTest, umfPoolTest,
     ::testing::Values(poolCreateExtParams{&MALLOC_POOL_OPS, nullptr,
                                           &UMF_NULL_PROVIDER_OPS, nullptr},
+                      poolCreateExtParams{&MALLOC_POOL_OPS, nullptr,
```

Comment:
I don't think this is needed - the MALLOC_POOL is implemented in a way such that it ignores the provider param and just calls malloc/free.

Reply:
Done. Removed.